testing/ncids-js-testing/config/webpack.dev.js

```diff
@@ -15,7 +15,7 @@ module.exports = merge(common, {
 	devServer: {
 		historyApiFallback: true,
 		static: paths.public,
-		open: true,
+		open: false,
```

Comment:
Did you mean to commit this? (Frankly I lothe the auto open, so I am all in favor of this)

Reply:
no but can i keep it? i'll feed it and wash it and pick up after it and pay for all the vet bills